UI/Result/index.js

```diff
@@ -36,8 +47,8 @@ export function Result({
                <h1>{computerName}</h1>
 
                <Card hasPlayerWon={!playerWon}>
-                  <Card.Image name={computersTurnCard.name} />
-                  <Card.Description description={computersTurnCard.cardDescription} />
+                  <Card.Image name={computersTurnCard.name}/>
+                  <Card.Description description={computersTurnCard.cardDescription}/>
                   <Card.AttributeList
                      attributes={GetAttributesFromCard(computersTurnCard)}
                      selectedAttribute={selectedAttribute}
```

Comment:
It's just the line below (55) that I can't comment on, but `hasPlayerWon={winner === PLAYER_COMPUTER}` should be used (in case of a draw)